src/Cli/ConfigGenerator.cs

```diff
@@ -114,6 +114,29 @@ public static bool TryCreateRuntimeConfig(InitOptions options, FileSystemRuntime
                 return false;
             }
 
+            bool isNestedInsertEnabledForGraphQL;
+
+            // Nested mutation operations are applicable only for MSSQL database. When the option --graphql.nested-insert.enabled is specified for other database types,
```

Comment:
nit: Please create a tracking issue to add support for other Sql types for nested create, and document it here

Reply:
I've created the issue : [#2001](https://github.com/Azure/data-api-builder/issues/2001) to track. Added a comment with the issue.